spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -293,17 +295,20 @@ def aggregate_per_slice_or_level(metric, mask=None, slices=[], levels=[], persli
             vertgroups = [tuple([level for level in levels])]
     # aggregation based on slices
     else:
-        vertgroups = None
         if perslice:
             # slicegroups = [(0,), (1,), (2,), (3,), (4,), (5,), (6,), (7,), (8,)]
             slicegroups = [tuple([slice]) for slice in slices]
         else:
             # slicegroups = [(0, 1, 2, 3, 4, 5, 6, 7, 8)]
             slicegroups = [tuple(slices)]
     agg_metric = dict((slicegroup, dict()) for slicegroup in slicegroups)
-
     # loop across slice group
     for slicegroup in slicegroups:
+        # add distance from PMJ info
+        if distance_pmj is not None:
+            agg_metric[slicegroup]['DistancePMJ'] = [distance_pmj]
```

Comment:
I'm a bit confused about the way we're using `distance_pmj` inside `aggregate_per_slice_or_level`. 

If I'm understanding correctly, we're just echoing the same `distance_pmj` value specified by the user, and we're not showing the offset for each slice, right?

Is that useful enough to deserve its own column in the CSV file?

Reply:
> If I'm understanding correctly, we're just echoing the same distance_pmj value specified by the user, and we're not showing the individual offset for each slice, right?

yes that's right

> Is that useful enough to deserve its own column in the CSV file? (Since it's the same value for every row?)

It would be useful if a person would run `sct_process_segmentation` with different distances and use the flag `-append`, so the CSV file would have various distances. Also, if we want to add this feature https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3432

Reply:
Ahhhhh! Okay, this makes much more sense. Especially now that I know that the distance is centered in the region, too.